espnet2/spk/loss/abs_loss.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/python
+# -*- encoding: utf-8 -*-
+# code from https://github.com/clovaai/voxceleb_trainer/blob/master/loss/aamsoftmax.py
+# Adapted from https://github.com/wujiyang/Face_Pytorch (Apache License)
```

Comment:
You can remove these lines.

Reply:
thanks, but I think it should remain because I copied the code from the comment webpage

Reply:
In this case, we should keep them and acknowledge the authors.

Reply:
I think it is OK to remove as this is the abstract class, not the actually function code.

Reply:
I see you also have the same comment in `espnet2/spk/loss/aamsoftmax.py`.